bin/minifollowups/pycbc_injection_minifollowup

```diff
@@ -119,30 +119,31 @@ if args.ifar_threshold is not None:
 num_events = int(workflow.cp.get_opt_tags('workflow-injection_minifollowups', 'num-events', ''))
 
 try:
-    if 'optimal_snr_1' in f['injections'].keys():  # 2-ifo pipeline case
-        optimal_snr = [f['injections/optimal_snr_1'][:][missed],
-                       f['injections/optimal_snr_2'][:][missed]]
-        # 'decisive' opt SNR is 2nd largest
-        dec_snr = numpy.minimum(optimal_snr[0], optimal_snr[1])
-    else:
-        optstrings = [os for os in f['injections'].keys() if \
-                      os.startswith('optimal_snr_')]
-        optimal_snr = [f['injections/%s' % os][:][missed] for os in optstrings]
-        # 2nd largest opt SNR
-        dec_snr = [sorted(snrs)[-2] for snrs in zip(*optimal_snr)]
-        dec_snr = numpy.array(dec_snr)
-    
+    optstrings = [os for os in f['injections'].keys() if \
+                  os.startswith('optimal_snr_')]
+    optimal_snr = [f['injections/%s' % os][:][missed] for os in optstrings]
+    # 2nd largest opt SNR
+    dec_snr = [sorted(snrs)[-2] for snrs in zip(*optimal_snr)]
+    dec_snr = numpy.array(dec_snr)
+
     if args.maximum_decisive_snr is not None:
         # By setting to 0, these injections will not be considered
         dec_snr[dec_snr > args.maximum_decisive_snr] = 0
     sorting = dec_snr.argsort()
     sorting = sorting[::-1]  # descending order of dec opt SNR
 except KeyError:
     # Fall back to effective distance if optimal SNR not available
-    eff_dist = {'H1': f['injections/eff_dist_h'][:][missed],
-                'L1': f['injections/eff_dist_l'][:][missed],
-                'V1': f['injections/eff_dist_v'][:][missed]
-                }
+    eff_dist = {}
+    for trig in single_triggers:
+        ifo = trig.ifo
+        eff_dist[ifo] = Detector(ifo).effective_distance(
+                             f['injections/distance'][:][missed],
+                             f['injections/ra'][:][missed],
+                             f['injections/dec'][:][missed],
+                             f['injections/polarization'][:][missed],
+                             f['injections/tc'][:][missed],
+                             f['injections/inclination'][:][missed])
```

Comment:
This does take vectorized input, right?

Reply:
The method can take vectorized input, yes.